client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -239,11 +239,15 @@ const InstalledInOrPricing = ( {
 	const isPluginActive = useSelector( ( state ) =>
 		getPluginOnSites( state, [ selectedSiteId ], plugin.slug )
 	)?.active;
-	const { isPreinstalledPremiumPlugin, isPreinstalledPremiumPluginUpgraded } =
-		usePreinstalledPremiumPlugin( plugin.slug );
-	const active = isWpcomPreinstalled || isPluginActive;
+	const {
+		isPreinstalledPremiumPlugin,
+		isPreinstalledPremiumPluginActive,
+		isPreinstalledPremiumPluginUpgraded,
+	} = usePreinstalledPremiumPlugin( plugin.slug );
+	const active = isWpcomPreinstalled || isPluginActive || isPreinstalledPremiumPluginActive;
 
-	let checkmarkColorClass = 'checkmark--active';
+	const checkmarkColorClass =
```

Comment:
This makes the check icon to appear as gray in the all sites view:

Before | After
--- | ---
<img width="478" alt="Screen Shot 2022-07-18 at 15 18 22" src="https://user-images.githubusercontent.com/1233880/179520105-eca87970-5eef-494d-ac53-6671dfce6da4.png"> | <img width="483" alt="Screen Shot 2022-07-18 at 15 18 27" src="https://user-images.githubusercontent.com/1233880/179520138-55b1db78-7dd4-43fe-b2e1-db97ce01fca0.png">

Is that intentional?

Reply:
Nope, regression, good catch! 🙇‍♂️